src/dash_tools.jl

```diff
@@ -378,9 +404,26 @@ function make_time_card()
 end
 
 """
-Retunrs a row containing a label, a tooltip and a filling box.
+Returns a row containing a label, a tooltip and a filling box.
 """
-function make_accordion_item(label::String="param", idx::String="id", msg::String="Message", value::Float64=1.0, min::Float64=1.0e-10, max::Float64=10_000.0)
+function make_accordion_item(label::String="param", idx::String="id", msg::String="Message", value::_T=1*one(_T), min=nothing, max=nothing) where _T <: Number
+    
+    if isa(_T, Float64)
```

Comment:
You could refactor this branching as something like 
```julia
 @inline _check_min(::Float64, ::Nothing) = 1e-10
 @inline _check_min(::Int64, ::nothing) = 2
 @inline _check_min(::T, x) where T = x
 @inline _check_max(::T, ::Nothing) where T = T(10_000)
 @inline _check_max(::T, x) where T = x

min = _check_min(_T, min)
max = _check_min(_T, max)
```

and get it all inlined